backend/controllers/user.controller.ts

```diff
@@ -21,10 +23,15 @@ import { Role } from '../../shared/user.enums';
 import { Inject } from 'typedi';
 import { GlobalsController } from './globals.controller';
 import { ApplicationsStatus } from '../../shared/globals.enums';
-import { Status } from '../../shared/app.enums';
+import { storageService } from '../services/storage.service.ts'
```

Comment:
Change from '../services/storage.service.ts' to --> '../services/storage.service'